### PR TITLE
DDF-2802 Add endpoints to jbake doc content

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

- completes list of endpoints for automated docs
- minor edits to templating process

#### Who is reviewing it?

@mcalcote @vinamartin @Lambeaux @jrnorth 

#### Select at least one member from relevant component team(s) from below.
[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl
@pklinef
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

- build succeeds
- all endpoints /services included in rendered docs

#### What are the relevant tickets?
[DDF-2802](https://codice.atlassian.net/browse/DDF-2802)

#### Screenshots (if appropriate)

[jdocumentation.pdf](https://github.com/codice/ddf/files/900922/jdocumentation.pdf)

#### Checklist:
- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

File: distribution/docs/pom.xml [hunk 1]
```diff
@@ -63,6 +63,7 @@
             <plugin>
                 <groupId>org.owasp</groupId>
                 <artifactId>dependency-check-maven</artifactId>
+                <version>1.2.9</version>
```
Comment (shaundmorris) [inline distribution/docs/pom.xml]: interesting why did you need to add this version here?

Comment (ricklarsen) [inline distribution/docs/pom.xml]: Double checking on that. I may have added it will tracking down a build issue and forgotten to take it back out.

Comment (ricklarsen) [inline distribution/docs/pom.xml]: removed
